### ISSUE TITLE
Update requirements around blog approval

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/community#1317]
### Affected Repository

https://github.com/open-telemetry/opentelemetry.io

### Requested changes

Please add more members to @open-telemetry/blog-approvers or remove the requirement to have an approval from that group for blog posts.

### Purpose

Each blog post needs an approval from that group. It would be good to have more people to potentially provide an approval for a blog post to share the load, or to drop that requirement and have docs-approvers/maintainers take care of reviews. 
 
### Expected Duration

None.

### Repository Maintainers

@open-telemetry/docs-maintainers

**Comment by arminru** [comment]
@open-telemetry/governance-committee Could you set up a guideline on how blog approvers are determined?
It might also be fine to have @open-telemetry/docs-maintainers as trusted members approve them instead.

**Comment by dyladan** [comment]
I'll add this to the agenda for the next GC meeting. There was no quorum last week and this week is cancelled so it will have to wait for the first week of january.

**Comment by svrnm** [comment]
> I'll add this to the agenda for the next GC meeting. There was no quorum last week and this week is cancelled so it will have to wait for the first week of january.

Thanks!

**Comment by dyladan** [comment]
GC meeting failed to get a quorum. Personally, I would be in support of delegating this to the comms sig, adding a team of communications maintainers and communications approvers, and letting them make whatever decision they feel is best.

**Comment by cartermp** [comment]
@dyladan Personally I am fine with having it deferred to us - in practice it more or less is, as blog posts usually only get held up by needing to get approval from the blog-owners group. I don't think we're at a point where we need more help reviewing the blog in addition to other areas either.

**Comment by trask** [comment]
> or remove the requirement to have an approval from that group (@open-telemetry/blog-approvers) for blog posts.

@svrnm where is this requirement documented or enforced? I support removing it also, just not sure where/how...

**Comment by austinlparker** [comment]
I don't think it was written down, it's an artifact from the Medium blog (when I migrated Medium -> GitHub, I simply re-applied the existing editors to that new group)

My 0.02 is that we don't need an official GC decision here because there was no GC decision in the first place. I'd be fine either removing @open-telemetry/blog-approvers ownership of that path or adding comms maintainers @svrnm and @cartermp to the group.

(From a least-privilege perspective I think there's probably a good argument to continue to maintain a separate group of approvers for blog posts, it's just that comms should own that group)

**Comment by austinlparker** [comment]
Helpfully enough I own @open-telemetry/blog-approvers so I've taken my own advice and added them to it.

**Comment by svrnm** [comment]
fixed
